### ISSUE TITLE
landing page

### ISSUE THREAD

**Issue by cjavdev** [discotroll65/vensei#3]
I like the idea of having a nice landing page especially for that legit start-up feel. that said i wouldn't spend too much time perfecting this. main focus should be on the battle ui :)

**Comment by discotroll65** [comment]
Definitely, I hear you -- Luckily, layout wize, it shouldn't be too hard to at least get the divs to just stack, with the right content in there, and move on ; probably won't even need to use backbone, except for maybe the navbar. Noted though :)
